src/index.css

```diff
@@ -168,6 +168,57 @@ body .container-body {
 .route-details .Select {
   margin-bottom: 20px;
 }
+
```

Comment:
It looks like you have duplicate code here.  Maybe check to see if there are any styles that need to be removed.